shared_requirements.txt

```diff
@@ -266,6 +266,8 @@ opentelemetry-sdk<2.0.0,>=1.5.0,!=1.10a0
 #override azure-mgmt-elasticsan msrest>=0.7.1
 #override azure-mgmt-eventgrid msrest>=0.7.1
 #override azure-ai-language-conversations isodate<1.0.0,>=0.6.1
+#override azure-ai-personalizer azure-core<2.0.0,>=1.24.0
+#override azure-ai-personalizer isodate<1.0.0,>=0.6.1
```

Comment:
you'll need to add typing-extensions here as well

Reply:
Fixed.